tests/foreman/cli/test_satellitesync.py

```diff
@@ -1163,6 +1163,91 @@ def test_negative_import_invalid_path(self, module_org, module_target_sat):
             '--metadata-file option'
         ) in error.value.message
 
+    @pytest.mark.tier3
+    @pytest.mark.parametrize(
+        'function_synced_rh_repo',
+        ['rhae2'],
+        indirect=True,
+    )
+    def test_negative_import_incomplete_archive(
+        self,
+        target_sat,
+        config_export_import_settings,
+        export_import_cleanup_function,
+        function_synced_rh_repo,
+        function_sca_manifest_org,
+        function_import_org_with_manifest,
```

Comment:
Do we need 2 fixtures which returns "org which has manifest uploaded"
fixture reference
`robottelo/pytest_fixtures/component/taxonomy.py` => `function_sca_manifest_org`
`robottelo/tests/foreman/cli/test_satellitesync.py` => `function_import_org_with_manifest`

Will it create 2 different orgs?

Reply:
It seems like it creates two different orgs. One is made to export in and the other is what we are importing into. @vsedmik correct me if I'm wrong. I am a ISS noob.

Reply:
Correct again! We need two orgs for that exact reason right now. I have a dream that we have a refactor and move to a completely separate import Satellite, one day.

Reply:
cool, ISS scenario can be perform on same satellite, good to know that.